Sources/RingPublishingTracking/Private/Tracking/PaidEventParameter.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PaidEventParameter.swift
+//  RingPublishingTracking
+//
+//  Created by Bernard Bijoch on 04/09/2024.
+//
+
+import Foundation
+
+enum PaidEventParameter: String {
```

Comment:
wybacz ale ja bym sie trzymał konwencji że jawnie przy budowaniu podajemy klucze
patrząc na to ja nie mam pojęcia co idzie do backendu, nie wiem jak to jest enkodowane

co ty na to by zrobić to tak jak do tej pory?